convert2rhel/unit_tests/conftest.py

```diff
@@ -101,34 +146,39 @@ def _pretend_fixture_base(monkeypatch, system_release_version):
     monkeypatch.setattr(
         utils,
         "get_file_content",
-        value=lambda _: "CentOS Linux release %s" % system_release_version,
+        value=lambda _: "%s release %s" % (system_name, system_version),
     )
     tool_opts.no_rpm_va = True
     system_info.resolve_system_info()
 
 
-def _pretend_centos_base(system_release_version):
-    """Factory function serves as a base to pretend_{os} fixtures."""
-
-    def fixture_centos7(monkeypatch, _replace_data_dir_for_centos7):
-        _pretend_fixture_base(monkeypatch, system_release_version)
-
-    def fixture_centos8(monkeypatch, _replace_data_dir_for_centos8):
-        _pretend_fixture_base(monkeypatch, system_release_version)
-
-    release2data_dir_relations = {
-        "8.3.2011": fixture_centos8,
-        "7.9.2009": fixture_centos7,
-    }
-
-    try:
-        return release2data_dir_relations[system_release_version]
-    except KeyError:
-        raise KeyError(
-            "Unknown system release version %s.\n"
-            "Available are: %s" % (system_release_version, " ".join(release2data_dir_relations)),
-        )
-
-
-pretend_centos8 = pytest.fixture(_pretend_centos_base("8.3.2011"))
-pretend_centos7 = pytest.fixture(_pretend_centos_base("7.9.2009"))
+all_systems = pytest.mark.parametrize(
+    "pretend_os",
+    (
+        ("7.9.1111", "CentOS Linux"),
```

Comment:
Not sure if need to bother with that, as we are not trying to replace the integration tests, but just giving more flexibility in unit testing. Let's leave it like it is until the need to have specific hotfix version (actually in this case it will be also possible just to write raw decorator instead of using aliases